### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by ionut1903**

### CODE REVIEW

File: src/app/HomeView/about-us/about-us.component.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div class="spContainer" id="aboutUsSection">
-    <app-section-title [title]="'Cu cine vei colabora?'"></app-section-title>   
+    <app-section-title [title]="'Cu cine vei colabora?'"></app-section-title>
```
Comment (colooo) [inline src/app/HomeView/about-us/about-us.component.html:2]: never put strings directly from html, set it as a variable

File: src/app/HomeView/home-header/home-header-small.css [hunk 2]
```diff
@@ -0,0 +1,10 @@
+@media all and (max-width: 768px) {
+  .homeHeaderContainer {
```
Comment (colooo) [inline src/app/HomeView/home-header/home-header-small.css:2]: always use kebab case for classes
https://en.wikipedia.org/wiki/Letter_case#cite_ref-30

File: src/app/HomeView/why-us/why-us.component.html [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <div class="spContainer" id="whyUsSection">
-    <app-section-title [title]="'De ce noi?'"></app-section-title> 
-    <div class="flexSpaceBtn">
+    <app-section-title [title]="'De ce noi?'"></app-section-title>
```
Comment (colooo) [inline src/app/HomeView/why-us/why-us.component.html:2]: move string to a variable

File: src/app/HomeView/why-us/why-us.component.ts [hunk 4]
```diff
@@ -5,7 +5,7 @@ import {MakeRequestService} from "../../services/make-request.service";
 @Component({
   selector: 'app-why-us',
   templateUrl: './why-us.component.html',
-  styleUrls: ['./why-us.component.css']
+  styleUrls: ['./why-us.component.css', './why-us-small-devices.css']
```
Comment (colooo) [inline src/app/HomeView/why-us/why-us.component.ts:8]: why not overwrite from styles.css?

File: src/app/app.module.ts [hunk 5]
```diff
@@ -3,8 +3,16 @@ import {NgModule} from '@angular/core';
...
+``
+//IMPORTS
+import {AngularFireModule} from '@angular/fire';
+import {AngularFireAnalyticsModule} from "@angular/fire/analytics";
+import {AngularFirestoreModule} from "@angular/fire/firestore";
 import {routing} from './sp.routing';
+import {firebaseConfig} from './environments/environment.dev';
 
+
+//DECLARATIONS
```
Comment (colooo) [inline src/app/app.module.ts:15]: also imports, not declarations :P

File: src/app/course-details/course-details-benefits/course-details-benefits.component.ts [hunk 6]
```diff
@@ -12,12 +13,18 @@ export class CourseDetailsBenefitsComponent implements OnInit {
   @Input() courseDetails: ICourseDetails;
   benefits: string[];
 title: any;
```
Comment (colooo) [inline src/app/course-details/course-details-benefits/course-details-benefits.component.ts:15]: set proper type, don't use any, string? HTMLElement? SAFEHtml?

File: src/app/course-details/course-details-confess-section/course-details-confess-section.component.ts [hunk 7]
```diff
@@ -12,12 +13,17 @@ export class CourseDetailsConfessSectionComponent implements OnInit {
   confessTextList: string[];
   title: any;
```
Comment (colooo) [inline src/app/course-details/course-details-confess-section/course-details-confess-section.component.ts:14]: same here, set type

File: src/app/course-details/course-details-confess-section/course-details-confess-section.component.ts [hunk 8]
```diff
@@ -12,12 +13,17 @@ export class CourseDetailsConfessSectionComponent implements OnInit {
...
   @Input() courseDetails: ICourseDetails;
-  constructor(private sanitizer: DomSanitizer) { }
+  constructor(private sanitizer: DomSanitizer,
+              private redirectService: RedirectService) { }
 
   ngOnInit(): void {
     this.title = this.sanitizer.bypassSecurityTrustHtml(this.courseDetails.courseConfessTitle);
```
Comment (colooo) [inline src/app/course-details/course-details-confess-section/course-details-confess-section.component.ts:20]: use sanitizer from the service that serves the data, always map the data from the service before subscribing to it

File: src/app/course-details/course-details-benefits/course-details-benefits.component.ts [hunk 9]
```diff
@@ -12,12 +13,18 @@ export class CourseDetailsBenefitsComponent implements OnInit {
...
-  constructor(private sanitizer: DomSanitizer) {
+  constructor(private sanitizer: DomSanitizer,
+              private redirectService: RedirectService) {
   }
 
   ngOnInit(): void {
     this.title = this.sanitizer.bypassSecurityTrustHtml(this.courseDetails.courseBenefitsTitle);
```
Comment (colooo) [inline src/app/course-details/course-details-benefits/course-details-benefits.component.ts:21]: use sanitizer from the service that serves the data, always map the data from the service before subscribing to it

File: src/app/course-details/course-details-benefits/course-details-benefits.component.ts [hunk 10]
```diff
@@ -12,12 +13,18 @@ export class CourseDetailsBenefitsComponent implements OnInit {
...
     this.benefits = this.courseDetails.courseBenefitsText.split(';');
     this.background = this.courseDetails.courseBenefitsPic[0]
```
Comment (colooo) [inline src/app/course-details/course-details-benefits/course-details-benefits.component.ts:23]: same here, map the data as needed from the service

File: src/app/course-details/course-details-confess-section/course-details-confess-section.component.ts [hunk 11]
```diff
@@ -12,12 +13,17 @@ export class CourseDetailsConfessSectionComponent implements OnInit {
...
     this.background = this.courseDetails.courseConfessBkPic[0];
     this.confessTextList = this.courseDetails.courseConfessText.split(';');
```
Comment (colooo) [inline src/app/course-details/course-details-confess-section/course-details-confess-section.component.ts:22]: map the data from the service

File: src/app/course-details/course-details-header/course-details-header.component.ts [hunk 12]
```diff
@@ -15,15 +16,16 @@ export class CourseDetailsHeaderComponent implements OnInit {
   background: string;
   @Input("checkIfLoggedIn") checkIfLoggedIn: EventEmitter<any> = new EventEmitter<any>();
```
Comment (colooo) [inline src/app/course-details/course-details-header/course-details-header.component.ts:17]: maybe the eventemitter uses type boolean?

File: src/app/course-details/course-details-header/course-details-header.component.ts [hunk 13]
```diff
@@ -15,15 +16,16 @@ export class CourseDetailsHeaderComponent implements OnInit {
...
   @Input() courseDetails: ICourseDetails;
-  constructor() {
+  constructor(private redirectService: RedirectService) {
 
   }
 
   ngOnInit(): void {
     this.background = this.courseDetails.headerBkPic[0];
```
Comment (colooo) [inline src/app/course-details/course-details-header/course-details-header.component.ts:24]: format the object from the service? 🗡️

File: src/app/course-details/course-details-how-it-works/course-details-how-it-works.component.ts [hunk 14]
```diff
@@ -10,14 +11,21 @@ import {DomSanitizer} from "@angular/platform-browser";
 export class CourseDetailsHowItWorksComponent implements OnInit {
 //todo: backgroun is realy neded?
   @Input() courseDetails: ICourseDetails;
-  background :string;
+  background: string;
   title: any;
```
Comment (colooo) [inline src/app/course-details/course-details-how-it-works/course-details-how-it-works.component.ts:15]: set type to SafeHtml

File: src/app/app.module.ts (hunk 5 above)
Comment (ionut1903) [reply src/app/app.module.ts:15]: This are declaration in ng module - is a comment to know to put there only the components...and the imports are imports in ng module...I know all are imports but this comment is only to let me be organized

**Pull request merged**